### PR TITLE
Add Adam's TowerViewer role to HTAN buckets

### PR DESCRIPTION
**Pull request by adamjtaylor**
This PR gives access to my  (Adam Taylor's) TowerViewer role for the HTAN buckets provisioned here.  
This is needed to facilitate outputs from Nextflow Tower runs into HTAN buckets.

### CODE REVIEW

**Comment by adamjtaylor** [comment]
FYI @clarisse-lau and @thomasyu888

**Comment by adamjtaylor** [comment]
@zaro0508 thank you in advance for approving and merging!

File: config/prod/htan-dcc-center-a.yaml [hunk 1]
```diff
@@ -20,11 +20,13 @@ parameters:
     - "arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/adam.taylor@sagebase.org"
     - "arn:aws:iam::728882028485:role/htan-project-TowerForgeBatchHeadJobRole-17MIZHWBA1TC5"
     - "arn:aws:iam::728882028485:role/htan-project-TowerForgeBatchWorkJobRole-17360PQARGWYU"
+    - "arn:aws:sts::728882028485:assumed-role/AWSReservedSSO_TowerViewer_fd6aee98ec127705/adam.taylor@sagebase.org"
   DenyDeleteARNs:
     - "arn:aws:iam::070278699608:user/inodb"
     - "arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/adam.taylor@sagebase.org"
     - "arn:aws:iam::728882028485:role/htan-project-TowerForgeBatchHeadJobRole-17MIZHWBA1TC5"
     - "arn:aws:iam::728882028485:role/htan-project-TowerForgeBatchWorkJobRole-17360PQARGWYU"
+    - "arn:aws:sts::728882028485:assumed-role/AWSReservedSSO_TowerViewer_fd6aee98ec127705/adam.taylor@sagebase.org"
```
Comment (zaro0508) [inline config/prod/htan-dcc-center-a.yaml:29]: this is deny you from deleting stuff in the bucket.  did you really want to restrict yourself from deleting stuff?

**Review by zaro0508 — APPROVED** [review]

File: config/prod/htan-dcc-center-a.yaml (hunk 1 above)
Comment (adamjtaylor) [reply config/prod/htan-dcc-center-a.yaml:29]: Yes, we have typically restricted delete for all users of HTAN buckets (including internal users) except for the two admin roles (@clarisse-lau  and @thomasyu888). Adding my TowerViewer role to DenyDelete was intentional.

**Pull request merged**